Salwador/ Models/ImageGenerationResponse.swift

```diff
@@ -10,3 +10,17 @@ struct ImageGenerationResponse: Decodable {
     let created: Int
```

Comment:
nit-pick: starting a folder name with a " " whitespace is in general discouraged.

Reply:
Good catch, just fixed the white space.